tests/Application/bin/console

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env php
 <?php
 
-use Tests\ThreeBRS\SyliusPaymentRestrictionPlugin\Application\Kernel;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
+use Tests\ThreeBRS\SyliusPaymentRestrictionPlugin\Kernel;
```

Comment:
protč tohle?

Reply:
Bez toho to padne
```
Fatal error: Uncaught Error: Class "Kernel" not found in /app/tests/Application/bin/console:36
Stack trace:
#0 {main}
  thrown in /app/tests/Application/bin/console on line 36
```
a protože chceme, aby test console použila test kernel.